tests/component/test_trigger_properties.py

```diff
@@ -0,0 +1,115 @@
+"""Contains a collection of pytest tests that validates trigger properties."""
+
+from nidaqmx.constants import TriggerType
+from nidaqmx.error_codes import DAQmxErrors
+from nidaqmx.errors import DaqError
+from nidaqmx.task import Task
+
+
+def test__trigger__get_float_property__returns_value(ai_voltage_chan_task: Task):
+    """Test to validate getter for trigger property of float type."""
+    # Default Rate of the pulse width filter timebase in NI PCIe-6363 device is 0.0
```

Comment:
If the test name says this is the default value, is this comment really needed? Same for the other tests that validate default values.

If you ask for an `ai_voltage_chan`, you shouldn't assume it comes from a PCIe-6363. 

If you make `ai_voltage_chan` a local fixture, it's okay to make more assumptions about what device it uses. However, `any_x_series_device` isn't guaranteed to give you a PCIe-6363.

Anyway, remove this comment, change the test name, and consider using a local fixture.

Reply:
Removed the comment